components/automate-chef-io/content/docs/iam-v1-overview.md

```diff
@@ -16,10 +16,11 @@ toc = true
 +++
 
 {{< warning >}}
-IAM v1 is deprecated and we recommend updating to the latest version of Automate to make use of [IAM v2]({{< relref "iam-v2-overview" >}}).
+This documentation and the feature it covers, IAM v1, are deprecated. This documentation will be removed soon. Most Chef Automate installations upgraded automatically to the current IAM implementation. Contact your customer service representative for help upgrading airgapped installations.
```

Comment:
This bit leaves me with a couple questions. These questions are probably more for engineering, but I think we should try to answer them and include them here:

- Do we know the version that the force upgrade will happen in? Many users do not have automatic upgrades turned on.  If I was an Automate users, I'd like a sentence here that was a bit more precise about what version this document applies to and how I check which one I need.

- Why do airgapped installations need to contact someone? We have an upgrade procedure for airgapped upgrades that users should be able to follow without assistance.  Are we anticipating problems here?

Reply:
>Do we know the version that the force upgrade will happen in

Not yet, but we can add the version when we do.

Reply:
refined phrasing based on @stevendanna's feedback.